src/vellum/workflows/nodes/core/retry_node/node.py

```diff
@@ -13,6 +15,37 @@
 
 
 class _RetryNodeMeta(BaseNodeMeta):
+    def __new__(cls, name: str, bases: Tuple[Type, ...], dct: Dict[str, Any]) -> Any:
```

Comment:
Most of this metaclass was copy pasted from `_TryNodeMeta`.

Before moving on to Map Node, going to consolidate this further to a common `BaseAdornableNode`